integreat_cms/cms/fixtures/test_data.json

```diff
@@ -248,7 +248,8 @@
       "admin_mail": "augsburg@integreat-app.de",
       "created_date": "2019-08-12T07:48:17.997Z",
       "last_updated": "2019-08-12T07:48:17.998Z",
-      "offers": [1, 2, 3]
+      "zammad_url": "https://zammad.augsburg.de",
```

Comment:
I would not use an existing domain for this purpose that we do not own. We could use zammad.example.com.

Reply:
Oh yes, fair point, I'll swap those out.